practice-one/src/styles/layouts/header.css

```diff
@@ -0,0 +1,38 @@
+.container-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 61px 0;
```

Comment:
Should use `padding-top: 61px` and `padding-bottom: 61px` instead use shorthand here